docs/user-guide/resource-browser.md

```diff
@@ -1,71 +1,438 @@
+## Discovering Resources 
 
-#### Events
+{% hint style="warning" %}
+### Who Can Perform This Action?
+Users need to have [access to the cluster](./global-configurations/authorization/user-access.md#kubernetes-resources-permissions) to discover resources.
+{% endhint %}
 
-An event is automatically generated in response to changes with other resources—like nodes, pods, or containers. For example, phases across a pod’s lifecycle transition from pending to running, or statuses like successful or failed may trigger a K8s event. The same goes for re-allocations and scheduling. These events are available until 15 minutes of deployment of the application.
+### Search and Filter
 
-#### Logs
+You can use the searchbox to browse the resources.
 
-Logs contain the logs of the Pods and Containers deployed which you can use for the process of debugging.
+![Figure 4: Locate Resources using Searchbox](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/kubernetes-resource-browser/discover-resource.gif)
 
+Moreover, you can use filters that allow you to quickly filter your workload as per labels, field selectors, or [CEL expression](https://kubernetes.io/docs/reference/using-api/cel/) as shown below.
 
-### Create Kubernetes Resource
+{% embed url="https://www.youtube.com/watch?v=E-V-ELCXtfs" caption="Filtering Workloads in Devtron" %}
 
-**Note**: As a pre-requisite, you must have a basic understanding of Kubernetes Cluster, Resources, Kinds.
+### Edit Manifest 
 
-You can create a Kubernetes resource by passing definition YAML file. You can create more than one resource by separating the resource YAMLs by ‘---’.
+{% hint style="warning" %}
+### Who Can Perform This Action?
+Users need to have super-admin permission to edit a manifest.
+{% endhint %}
 
-An example that shows the required fields and object specifications for a Kubernetes Deployment:
+You can edit the [manifest](../reference/glossary.md#manifest) of a Kubernetes object. This can be for fixing errors, scaling resources, or changing configuration.
+
+![Figure 5: Editing a Live Manifest](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/kubernetes-resource-browser/edit-live-manifest.gif)
+
+### View Events
+
+You can monitor activities like creation, deletion, updation, scaling, or errors in the resources involved.
+
+![Figure 6: Viewing All Events](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/kubernetes-resource-browser/events.gif)
+
+### Delete
+
+{% hint style="warning" %}
+### Who Can Perform This Action?
+Users need to have super-admin permission to delete a resource.
+{% endhint %}
+
+You can delete an unwanted resource if it is orphaned and no longer required by your applications.
+
+![Figure 7: Deleting a Resource](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/kubernetes-resource-browser/delete.gif)
+
+You can perform all the actions on the following:
```

Comment:
Why mention this? User can do on all kubernetes objects...

Reply:
Removed